.gitignore

```diff
@@ -46,3 +46,5 @@ dist/
 node_modules/
 yarn.lock
 package-lock.json
+
+.history/
```

Comment:
Let's not add such things to our forks. Fine to have it in project repos, but if we'd want to sync this with upstream at some point, extensions might not be accepted.

Reply:
@SergeiGrigorev remove this please before pushing.

Reply:
Done